src/libraries/System.Numerics.Tensors/src/System/Numerics/Tensors/TensorPrimitives.netstandard.cs

```diff
@@ -640,11 +640,14 @@ private static unsafe Vector<float> LoadRemainderMaskSingleVector(int validItems
         {
             Debug.Assert(Vector<float>.Count is 4 or 8 or 16);
 
-            return AsVector(
-                ref Unsafe.As<uint, float>(ref MemoryMarshal.GetReference(RemainderUInt32Mask_16x16)),
-                (validItems * 16) + (16 - Vector<float>.Count));
+            return Vector.ConditionalSelect(
+                (Vector<float>)Vector.GreaterThan(AsVector(ref MemoryMarshal.GetReference<int>(s_0through15)), new Vector<int>(Vector<int>.Count - 1 - validItems)),
+                ~Vector<float>.Zero,
+                Vector<float>.Zero);
         }
 
+        private static readonly int[] s_0through15 = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
```

Comment:
Does it need to be a `ROS<int>` ?

Reply:
This is in the netstandard build where CreateSpan doesn't exist. As a result, if this returns a span, Roslyn will generate a lazily-initialized array.